.gitignore

```diff
@@ -1,20 +1,29 @@
 PyRoute/__pycache__/
+PyRoute/Allies/__pycache__/
```

Comment:
You can do this better with the pattern `**/__pycache__/` rather than listing out all of the directories one by one.

Reply:
Awright, I've made that change - you were right, it does clean up `.gitignore` a lot.